install_config/install/prerequisites.adoc

```diff
@@ -80,12 +80,12 @@ ifdef::openshift-origin[]
 installation option, or RHEL Atomic Host 7.2.4 or later.
 endif::[]
 ifdef::openshift-enterprise[]
-- Base OS: RHEL 7.1 or later with "Minimal" installation option, or RHEL Atomic
+- Base OS: RHEL 7.2 or later with "Minimal" installation option, or RHEL Atomic
 Host 7.2.4 or later.
 endif::[]
 - 2 vCPU.
-- Minimum 8 GB RAM.
-- Minimum 30 GB hard disk space for the file system containing *_/var/_*.
+- Minimum 16 GB RAM.
+- Minimum 40 GB hard disk space for the file system containing *_/var/_*.
```

Comment:
We've had OOM on small clusters that have been OOMing in the field.  This is the base case, hence minimum.